infogami/infobase/common.py

```diff
@@ -128,7 +128,7 @@ def create_test_store():
     >>> t
     <thing: u'/type/type'>
     >>> t.properties[0]
-    <Storage {'expected_type': <thing: u'/type/string'>, 'unique': True, 'name': 'name'}>
+    <Storage {'expected_type': <thing: u'/type/string'>, 'name': 'name', 'unique': True}>
```

Comment:
I believe that this will doctest will now fail on Python 2.

Reply:
Yeah -- the test suite agrees with you. :) See revised push!